### PR TITLE
Expose Identity Profiles in search.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: mozillians/users/search_indexes.py [hunk 1]
```diff
@@ -53,3 +53,19 @@ def prepare_groups(self, obj):
     def index_queryset(self, using=None):
         """Exclude incomplete profiles from indexing."""
         return self.get_model().objects.complete()
+
+
+class IdpProfileIndex(indexes.SearchIndex, indexes.Indexable):
+    """IdpProfile Profile Search Index."""
+    # Primary field of the index
+    text = indexes.CharField(document=True, use_template=True)
```
[the remark this answers is not in the document]
Comment (akatsoulas) [reply mozillians/users/search_indexes.py]: I believe it is. The only downside probably is that we won't be able to add additional fields if we ever need them.

[the remark this answers is not in the document]
Comment (fiji-flo) [reply mozillians/users/search_indexes.py]: Indeed

**Comment by akatsoulas** [comment]
@fiji-flo PR updated. I solved the duplicate profiles issu by not indexing the IdpProfile of a user if the email already exists in the UserProfile index. This is handled in the `prepare_idp_email` method.

File: mozillians/phonebook/views.py [hunk 2]
```diff
@@ -522,10 +522,6 @@ class PhonebookSearchView(SearchView):
     form_class = forms.PhonebookSearchForm
     template_name = 'phonebook/search.html'
 
-    def get_queryset(self):
```
Comment (akatsoulas) [inline mozillians/phonebook/views.py]: @fiji-flo fyi, I also dropped this method because we don't do anything more than calling the `super`.

**Pull request merged**